model/lifecycle.go

```diff
@@ -55,9 +55,6 @@ type VersionToRestart struct {
 // SetVersionActivation updates the "active" state of all builds and tasks associated with a
 // version to the given setting. It also updates the task cache for all builds affected.
 func SetVersionActivation(versionId string, active bool, caller string) error {
```

Comment:
Doesn't seem like we need this function at all since it just calls setTaskActivationForVersion. Should we just remove that private function and move the logic in here?